### ISSUE TITLE
All Add-ons refusing to Start

### ISSUE THREAD

**Issue by AndyinHawick** [home-assistant/supervisor#4105]
### Describe the issue you are experiencing

I have now ported my HA onto a Pi3 set up solely to run HA (rather than the rather messy containerised version that I was running before). 
HA 2023.1.4
Supervisor 2022.12.1
Operating System 9.4
Frontend 20230110.0 - latest

None of the Add-ons now work.
File editor
SSH
Mosquitto
Studio Code Server

I have tried uninstalling and re-installing, restarting the computer all to no avail.

I click the Start button and, the first time, it 'whirs' as if it is doing something but then comes back to 'Start' still showing and the run indicator remaining red.

When I select File Editor from the sidebar, it comes up with a splash screen stating that the add-on is not running, "Please start it first".

Is this a bug or am I missing something?

Thanks
   Andy

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

File editor

### What is the version of the add-on?

5.4.2

### Steps to reproduce the issue

1. Settings > Add-ons
2. File-editor (or any other add-on)
3. START
4. 
...

## System Information

version | core-2023.1.4
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.10.7
os_name | Linux
os_version | 5.15.76-v8
arch | aarch64
timezone | Europe/London
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.29.1
Stage | running
Available Repositories | 1266
Downloaded Repositories | 17

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 9.4
-- | --
update_channel | stable
supervisor_version | supervisor-2022.12.1
agent_version | 1.4.1
docker_version | 20.10.19
disk_total | 27.7 GB
disk_used | 3.9 GB
healthy | true
supported | true
board | rpi3-64
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.4.2)

</details>

<details><summary>Dashboards</summary>

dashboards | 3
-- | --
resources | 5
views | 11
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 2 January 2023 at 10:02
-- | --
current_recorder_run | 16 January 2023 at 19:30
estimated_db_size | 823.37 MiB
database_engine | sqlite
database_version | 3.38.5

</details>

### Anything in the Supervisor logs that might be useful for us?

```txt
23-01-16 19:30:06 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-01-16 19:30:09 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
23-01-16 19:30:09 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
23-01-16 19:30:09 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/esphome/home-assistant-addon repository
23-01-16 19:30:09 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-01-16 19:30:14 INFO (MainThread) [supervisor.store] Loading add-ons from store: 68 all - 0 new - 0 remove
23-01-16 19:30:14 INFO (MainThread) [supervisor.store] Loading add-ons from store: 68 all - 0 new - 0 remove
23-01-16 19:30:38 INFO (MainThread) [supervisor.auth] Auth request from 'core_mosquitto' for 'mosquito'
23-01-16 19:30:40 INFO (MainThread) [supervisor.auth] Home Assistant not running, checking cache
23-01-16 19:30:49 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-01-16 19:32:39 INFO (MainThread) [supervisor.host.info] Updating local host information
23-01-16 19:32:42 INFO (MainThread) [supervisor.host.services] Updating service information
23-01-16 19:32:42 INFO (MainThread) [supervisor.host.network] Updating local network information
23-01-16 19:32:42 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
23-01-16 19:32:43 INFO (MainThread) [supervisor.host.manager] Host information reload completed
23-01-16 19:33:03 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
23-01-16 19:38:34 WARNING (MainThread) [supervisor.addons.addon] a0d7b954_vscode is already running!
23-01-16 19:38:37 WARNING (MainThread) [supervisor.addons.addon] a0d7b954_vscode is already running!
23-01-16 19:38:38 WARNING (MainThread) [supervisor.addons.addon] a0d7b954_vscode is already running!
23-01-16 19:38:49 WARNING (MainThread) [supervisor.addons.addon] core_configurator is already running!
23-01-16 19:38:58 INFO (SyncWorker_5) [supervisor.docker.interface] Stopping addon_core_configurator application
23-01-16 19:39:04 INFO (SyncWorker_5) [supervisor.docker.interface] Cleaning addon_core_configurator application
23-01-16 19:39:04 INFO (SyncWorker_5) [supervisor.docker.interface] Removing image homeassistant/aarch64-addon-configurator with latest and 5.4.2
23-01-16 19:39:06 INFO (MainThread) [supervisor.addons.addon] Removing add-on data folder /data/addons/data/core_configurator
23-01-16 19:39:07 INFO (MainThread) [supervisor.ingress] Update Ingress as panel for core_configurator
23-01-16 19:39:07 INFO (MainThread) [supervisor.addons] Add-on 'core_configurator' successfully removed
23-01-16 19:39:10 INFO (SyncWorker_2) [supervisor.docker.interface] Stopping addon_a0d7b954_vscode application
23-01-16 19:39:17 INFO (SyncWorker_2) [supervisor.docker.interface] Cleaning addon_a0d7b954_vscode application
23-01-16 19:39:18 INFO (SyncWorker_2) [supervisor.docker.interface] Removing image ghcr.io/hassio-addons/vscode/aarch64 with latest and 5.5.1
23-01-16 19:39:31 INFO (MainThread) [supervisor.addons.addon] Removing add-on data folder /data/addons/data/a0d7b954_vscode
23-01-16 19:39:31 INFO (MainThread) [supervisor.ingress] Update Ingress as panel for a0d7b954_vscode
23-01-16 19:39:32 INFO (MainThread) [supervisor.addons] Add-on 'a0d7b954_vscode' successfully removed
23-01-16 19:39:32 WARNING (MainThread) [supervisor.addons.addon] core_mosquitto is already running!
23-01-16 19:39:42 WARNING (MainThread) [supervisor.addons.addon] core_mosquitto is already running!
23-01-16 19:40:12 WARNING (MainThread) [supervisor.addons.addon] a0d7b954_ssh is already running!
23-01-16 19:41:45 WARNING (MainThread) [supervisor.addons.addon] a0d7b954_ssh is already running!
23-01-16 19:41:51 WARNING (MainThread) [supervisor.addons.addon] a0d7b954_ssh is already running!
23-01-16 19:43:00 INFO (SyncWorker_0) [supervisor.docker.interface] Stopping addon_a0d7b954_ssh application
23-01-16 19:43:05 INFO (SyncWorker_0) [supervisor.docker.interface] Cleaning addon_a0d7b954_ssh application
23-01-16 19:43:06 INFO (SyncWorker_0) [supervisor.docker.interface] Removing image ghcr.io/hassio-addons/ssh/aarch64 with latest and 13.0.0
23-01-16 19:43:10 INFO (MainThread) [supervisor.addons.addon] Removing add-on data folder /data/addons/data/a0d7b954_ssh
23-01-16 19:43:10 INFO (MainThread) [supervisor.ingress] Update Ingress as panel for a0d7b954_ssh
23-01-16 19:43:10 INFO (MainThread) [supervisor.addons] Add-on 'a0d7b954_ssh' successfully removed
23-01-16 19:43:36 WARNING (MainThread) [supervisor.addons.addon] core_mariadb is already running!
23-01-16 19:43:39 INFO (SyncWorker_5) [supervisor.docker.interface] Stopping addon_core_mariadb application
23-01-16 19:43:47 INFO (SyncWorker_5) [supervisor.docker.interface] Cleaning addon_core_mariadb application
23-01-16 19:43:47 INFO (SyncWorker_5) [supervisor.docker.interface] Removing image homeassistant/aarch64-addon-mariadb with latest and 2.5.1
23-01-16 19:43:49 INFO (MainThread) [supervisor.addons.addon] Removing add-on data folder /data/addons/data/core_mariadb
23-01-16 19:43:49 INFO (MainThread) [supervisor.addons] Add-on 'core_mariadb' successfully removed
23-01-16 19:44:05 WARNING (MainThread) [supervisor.addons.addon] core_mosquitto is already running!
23-01-16 19:44:13 INFO (SyncWorker_0) [supervisor.docker.interface] Stopping addon_core_mosquitto application
23-01-16 19:44:19 INFO (SyncWorker_0) [supervisor.docker.interface] Cleaning addon_core_mosquitto application
23-01-16 19:44:19 INFO (SyncWorker_0) [supervisor.docker.interface] Removing image homeassistant/aarch64-addon-mosquitto with latest and 6.1.3
23-01-16 19:44:22 INFO (MainThread) [supervisor.addons.addon] Removing add-on data folder /data/addons/data/core_mosquitto
23-01-16 19:44:22 INFO (MainThread) [supervisor.discovery] Delete discovery to Home Assistant mqtt from core_mosquitto
23-01-16 19:44:22 INFO (MainThread) [supervisor.addons] Add-on 'core_mosquitto' successfully removed
23-01-16 19:44:22 INFO (MainThread) [supervisor.discovery] Discovery 53bd7a25dc944f39bcf5cdb04ec39440 message send
23-01-16 20:02:44 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-01-16 20:12:38 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
23-01-16 20:12:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.IPV4_CONNECTION_PROBLEM/ContextType.SYSTEM
23-01-16 20:12:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.NO_CURRENT_BACKUP/ContextType.SYSTEM
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.check] System checks complete
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
23-01-16 20:12:39 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
23-01-16 20:16:10 INFO (MainThread) [supervisor.addons] Creating Home Assistant add-on data folder /data/addons/data/core_configurator
23-01-16 20:16:10 INFO (SyncWorker_5) [supervisor.docker.interface] Downloading docker image homeassistant/aarch64-addon-configurator with tag 5.4.2.
23-01-16 20:17:12 INFO (MainThread) [supervisor.addons] Add-on 'core_configurator' successfully installed
23-01-16 20:17:19 INFO (SyncWorker_3) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-configurator with version 5.4.2
23-01-16 20:20:15 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-01-16 20:20:15 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/esphome/home-assistant-addon repository
23-01-16 20:20:15 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
23-01-16 20:20:15 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
23-01-16 20:20:20 INFO (MainThread) [supervisor.store] Loading add-ons from store: 68 all - 0 new - 0 remove
23-01-16 20:20:20 INFO (MainThread) [supervisor.store] Loading add-ons from store: 68 all - 0 new - 0 remove
23-01-16 20:22:06 WARNING (MainThread) [supervisor.addons.addon] core_configurator is already running!
23-01-16 20:27:36 WARNING (MainThread) [supervisor.addons.addon] core_configurator is already running!
23-01-16 20:32:45 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
```

### Anything in the add-on logs that might be useful for us?

```txt
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun configurator (no readiness notification)
s6-rc: info: service legacy-services successfully started
INFO:2023-01-16 20:17:26,031:hass_configurator.configurator:Starting server
INFO:2023-01-16 20:17:26,033:hass_configurator.configurator:Listening on: http://0.0.0.0:8099
```

### Additional information

All of these were working under the previous installation and even after reinstalling and restarting they fail to launch.

**Comment by WvanWaas** [comment]
Looks like I'm in the same boat my api calls for OverKizz, Eufy Alarm and even XBOX are failing due to an error from add-ons. The add on page isn't opening anymore. Something is broken since yesterday.

**Comment by mdegat01** [comment]
Hm supervisor did not update yesterday on any channel so not sure how that could be.

Lets try a simple test to start. Please uninstall File Editor and reboot the machine ([Settings -> System -> Hardware](https://my.home-assistant.io/redirect/hardware/), click the 3 dots in the top right and select "Reboot System"). Then please try to install and start file editor after HA starts up again. If that doesn't work please share the logs.

If that does work then try to do a partial restore of the backup and just restore the SSH, Mosquitto, and Studio Code Server. If that works then great and if not then please share the supervisor logs. We can also try a reinstall approach for these but since I assume these addons actually have config and potentially data hopefully we can get a restore to work for these.

**Comment by WvanWaas** [comment]
I have been trying to uninstall the file editor but wasn't able to access the file editor, Add ons and backup pages. I have saved the logs but cleared the SD Card and placed an backup from friday in my install. Lost some data but up and running again.

**Comment by AndyinHawick** [comment]
Hi Mike

Thanks for your reply.

I had earlier gone through the sequence of uninstalling all of the add-ons, restarting HA and re-installing add-ons one at a time but the outcome was almost always the same.

I did manage to successfully install Samba and gain access to the files through that route but Terminal/SSH and File Editor remained stubbornly not running.

I noted an update to File Editor, which I applied. I also powered down the Pi this morning for other reasons and, having powered it back up, I now discover that both File Editor and Terminal are functioning.

Not sure quite what made the difference but it's good to have them back! Whether it was something linked to my porting to the Pi or whether it was systemic as someone else noted, I am not sure.

Cheers
	Andy
